api-admin/src/main/java/bio/terra/pearl/api/admin/service/forms/ConsentFormExtService.java

```diff
@@ -17,38 +18,76 @@ public class ConsentFormExtService {
   private AuthUtilService authUtilService;
   private ConsentFormService consentFormService;
   private StudyEnvironmentConsentService studyEnvironmentConsentService;
+  private StudyEnvironmentService studyEnvironmentService;
 
   public ConsentFormExtService(
       AuthUtilService authUtilService,
       ConsentFormService consentFormService,
-      StudyEnvironmentConsentService studyEnvironmentConsentService) {
+      StudyEnvironmentConsentService studyEnvironmentConsentService,
+      StudyEnvironmentService studyEnvironmentService) {
     this.authUtilService = authUtilService;
     this.consentFormService = consentFormService;
     this.studyEnvironmentConsentService = studyEnvironmentConsentService;
+    this.studyEnvironmentService = studyEnvironmentService;
   }
 
   public ConsentForm createNewVersion(
       String portalShortcode, ConsentForm consentForm, AdminUser user) {
-    if (!user.isSuperuser()) {
-      throw new PermissionDeniedException("You do not have permissions to perform this operation");
-    }
     Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
     return consentFormService.createNewVersion(portal.getId(), consentForm);
   }
 
+  public ConsentForm create(String portalShortcode, ConsentForm consentForm, AdminUser user) {
+    Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
+    consentForm.setPortalId(portal.getId());
+    return consentFormService.create(consentForm);
+  }
+
   public StudyEnvironmentConsent updateConfiguredConsent(
       String portalShortcode,
       EnvironmentName envName,
+      String studyShortcode,
       StudyEnvironmentConsent updatedObj,
       AdminUser user) {
-    authUtilService.authUserToPortal(user, portalShortcode);
-    if (!EnvironmentName.sandbox.equals(envName)) {
-      throw new IllegalArgumentException(
-          "Updates can only be made directly to the sandbox environment".formatted(envName));
-    }
+    authConfiguredConsentRequest(portalShortcode, envName, studyShortcode, updatedObj, user);
     StudyEnvironmentConsent existing =
         studyEnvironmentConsentService.find(updatedObj.getId()).get();
     BeanUtils.copyProperties(updatedObj, existing);
     return studyEnvironmentConsentService.update(existing);
   }
+
+  public StudyEnvironmentConsent createConfiguredConsent(
+      String portalShortcode,
+      EnvironmentName envName,
+      String studyShortcode,
+      StudyEnvironmentConsent newObj,
+      AdminUser user) {
+    authConfiguredConsentRequest(portalShortcode, envName, studyShortcode, newObj, user);
```

Comment:
"ConfiguredConsent" and "ConfiguredSurvey" are used in various places in the code as a more readable shorthand for "SutdyEnvironmentConsent" and "StudyEnvironmentSurvey", since they apply studyEnv-specific configuration to a survey/consent.  If I had to do it all over again, StudyEnvironmentConsent would probably just be called "ConfiguredConsent"